controller/users.go

```diff
@@ -49,6 +50,102 @@ func NewUsersController(service *goa.Service, db application.DB, config UsersCon
 		config:     config,
 	}
 }
+func randString(n int) string {
+	const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letterBytes[rand.Int63()%int64(len(letterBytes))]
+	}
+	return string(b)
+}
+
+// Obfuscate runs the obfuscate action to soft delete sensitive data associated
+// with an user and her associated identity.
+func (c *UsersController) Obfuscate(ctx *app.ObfuscateUsersContext) error {
+	isSvcAccount, err := isServiceAccount(ctx, serviceNameAuth)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to determine if account is a service account")
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err))
+
+	}
+	if !isSvcAccount {
+		log.Error(ctx, map[string]interface{}{
+			"identity_id": ctx.ID,
+		}, "account used to call create api is not a service account")
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(errs.New("a non-service account tried to create a user.")))
+	}
+	u, err := uuid.FromString(ctx.ID)
```

Comment:
Can we use a better variable name, please?
```suggestion
	userID, err := uuid.FromString(ctx.ID)
```

Reply:
+1